### PR TITLE
feat(healthCheck): Optimize logger and parameter loading

### PR DESCRIPTION
**Pull request by benjaminParisel**
* Add tests mecanism
* Parse parameter on starting
* Be available to start server with config file
* Add Logger

### CODE REVIEW

File: src/server/ManageParameters.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+const fs = require('fs');
+
+function getParameter(param) {
+  return param.substr(param.indexOf('=') + 1);
+}
```
Comment (jeromecambon) [inline src/server/ManageParameters.js]: This function should be in the ManageParameters class

File: src/server/StudioHealthCheck.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+const request = require('request');
+const winston = require('winston');
+
+/**
+ * @type {module.StudioHealthCheck}
+ */
+module.exports = class StudioHealthCheck {
```
Comment (jeromecambon) [inline src/server/StudioHealthCheck.js]: To be consistent with other classes, 'module.exports' fter the class

File: src/server/bdrServer.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+/**
+ * Attach and call HealthCheck if parameter has attach for server parameter
+ * @constructor
+ */
+function DoHealthCheck(config) {
+  if (config.healthCheckUrl && config.healthCheckPort) {
+    logger.info('Listen studio health check connection');
+    let studioHealthCheck = new StudioHealthCheck(config.healthCheckUrl, config.healthCheckPort);
+    studioHealthCheck.healthCheckWithInterval(2000);
```
Comment (jeromecambon) [inline src/server/bdrServer.js]: Every 2s seems a bit too much (?). Is it a Studio recommendation? I would use 10s (or more)

File: src/server/bdrServer.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+ */
+server.start({ port: port, endpoint: '/repository' }, () => {
+  logger.info(
+    '\n' +
+      '________          __                                                      .__  __                       \n' +
+      '\\______ \\ _____ _/  |______            _______   ____ ______   ____  _____|__|/  |_  ___________ ___.__.\n' +
+      ' |    |  \\\\__  \\\\   __\\__  \\    ______ \\_  __ \\_/ __ \\\\____ \\ /  _ \\/  ___/  \\   __\\/  _ \\_  __ <   |  |\n' +
+      ' |    `   \\/ __ \\|  |  / __ \\_ /_____/  |  | \\/\\  ___/|  |_> >  <_> )___ \\|  ||  | (  <_> )  | \\/\\___  |\n' +
+      '/_______  (____  /__| (____  /          |__|    \\___  >   __/ \\____/____  >__||__|  \\____/|__|   / ____|\n' +
+      '        \\/     \\/          \\/                       \\/|__|              \\/                       \\/     \n'
```
Comment (jeromecambon) [inline src/server/bdrServer.js:88]: Do we really need this ?

File: src/server/bdrServer.js (hunk 3 above)
Comment (benjaminParisel) [reply src/server/bdrServer.js]: Good catch i put 200 for test

File: src/server/bdrServer.js (hunk 4 above)
Comment (benjaminParisel) [reply src/server/bdrServer.js:88]: Yeah ! It's beautiful

**Pull request merged**